### PR TITLE
type(performance): ⚡️ Optimized Scaffold and increased Performance⚡️

### PR DESCRIPTION
**Pull request by S-ayanide**
Signed-off-by: Sayan Mondal <sayan@chaosnative.com>

## Proposed changes

* Moved Scaffold higher up in the hierarchy out of Router to stop re-render
* Removed Breadcrumb from the Header
* Added decodeURI to decode URLs better

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [X] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by vanshBhatia-A4k9 — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

File: litmus-portal/frontend/src/containers/app/App.tsx [hunk 1]
```diff
@@ -114,144 +113,144 @@ const Routes: React.FC = () => {
...
+              <Redirect
+                exact
+                path="/analytics/litmusdashboard"
+                to="/analytics"
+              />
+              <Redirect
+                exact
+                path="/analytics/kubernetesdashborad"
+                to="/analytics"
+              />
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/containers/app/App.tsx:246]: this one may be removed

**Review by amityt — APPROVED** [review]

File: litmus-portal/frontend/src/containers/app/App.tsx (hunk 1 above)
Comment (S-ayanide) [reply litmus-portal/frontend/src/containers/app/App.tsx:246]: Sure

Comment (S-ayanide) [reply litmus-portal/frontend/src/containers/app/App.tsx]: Removed this

**Review by whitetiger1399 — APPROVED** [review]
Thanks for these enhancements

**Pull request merged**